src/src/components/ObjectBrowser.jsx

```diff
@@ -3632,13 +3665,40 @@ class ObjectBrowser extends Component {
                         />}
                         label={this.props.t('ra_Do not export values of states')}
                     />
+                    <br />
+                    {this.props.t('ra_These options can reduce the size of the export file:')}
+                    <FormControlLabel
+                        control={<Checkbox
+                            checked={this.state.beautifyJsonExport}
+                            onChange={e => this.setState({ beautifyJsonExport: e.target.checked })}
+                        />}
+                        label={this.props.t('ra_Beautify JSON output')}
+                    />
+                    <br />
+                    <FormControlLabel
+                        control={<Checkbox
+                            checked={this.state.excludeSystemRepositoriesFromExport}
+                            onChange={e => this.setState({ excludeSystemRepositoriesFromExport: e.target.checked })}
+                        />}
+                        label={this.props.t('ra_Exclude system repositories from export JSON')}
+                    />
+                    <FormControlLabel
+                        control={<Checkbox
+                            checked={this.state.excludeTranslations}
+                            onChange={e => this.setState({ excludeTranslations: e.target.checked })}
+                        />}
+                        label={this.props.t('ra_Exclude translations (except english) from export JSON')}
```

Comment:
you can find the translations here: https://github.com/ioBroker/ioBroker.admin/tree/master/src/src/i18n

Please do not use the `ra_` prefix for the translations managed in this repository.

Reply:
I saw that folder, but as some tools/scripts are used in adapters to populate translations I wasn't sure whether to add it there manually.

@foxriver76 I added translations for german and english, is there a tool/automatism which can fill the other languages at the correct position as well, or do I have to manually fill those?

Reply:
We are normally using https://github.com/gilmarsquinelato/i18n-manager but needs Google API key. I can add the rest of the translations after merge, no problem.